### PR TITLE
ci: remove Tarantool tests from LuaJIT integration

### PR DESCRIPTION
**Pull request by mkokryashkin**
After the commit 3daf2399dd5b ('ci: fix step parameters for reusable runs') it is now possible to create Tarantool integration workflows for any repository in the Tarantool organization. Considering this, we don't need to run Tarantool tests under the old target for the LuaJIT integration in .test.mk and we can leave only LuaJIT tests in this target for the sake of exotic Tarantool builds testing.

NO_DOC=CI
NO_TEST=CI
NO_CHANGELOG=CI

### CODE REVIEW

**Review by igormunkin — APPROVED** [review]
@mkokryashkin, thanks for the patch! LGTM.

Side note: AFAIU, this patch is blocked by the series related to LuaJIT CI, so please add `do not merge` label, until the integration CI refactoring lands the long-term branches in LuaJIT repo.

**Review by ochaplashkin — APPROVED** [review]

**Pull request merged**
